### PR TITLE
Adding resource

### LINKED ISSUE #245: 💡 [FEATURE] - Adding a resource under web development Readme.md file
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

I would like to contribute to Readme.md file under web development by adding a playlist under YouTube Channels

### Explain your solution

I would like to add the playlist of Apna college which I found was very helpful for beginners

### Any alternative approaches/features

NA

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by SaileePhal**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - YouTube Channel`

### Fixes Issue
Closes #245 

### Description: 
Added the resource of Apna College under the YouTube Channel section in Web Development Readme.md File

### Resource URL: 
Apna College: https://youtube.com/playlist?list=PLfqMhTWNBTe3H6c9OGXb5_6wcc1Mca52n&si=UYYv7L3NHDF31bHL

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

**Comment by jfmartinz** [comment]
Is this course about front end, backend or full stack?

**Comment by jfmartinz** [comment]
Add it [here](https://github.com/jfmartinz/ResourceHub/tree/main/Web%20Development#front-end-development-1) instead, nad I will merged it after this change, thanks

**Comment by SaileePhal** [comment]
Sure sir. Will do the necessary changes suggested by you.

**Comment by jfmartinz** [comment]
Solve the conflict

**Comment by SaileePhal** [comment]
Sure sit on it.

**Comment by SaileePhal** [comment]
Done sir via #326

**Comment by jfmartinz** [comment]
Please solve the  conflict before I merged this pr, you can refer to this [link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) for more information

File: Web Development/README.md [hunk 1]
```diff
@@ -1312,6 +1312,11 @@ to everyone - everywhere!</td>
         <td>Youtube channel that teaches Web Development in a super simple way.Explore the content rich channel to become a pro in web development.
         </td>
       </tr>
+      <tr>
+        <td><a href="https://youtube.com/playlist?list=PLfqMhTWNBTe3H6c9OGXb5_6wcc1Mca52n&si=UYYv7L3NHDF31bHL">Apna College</a></td>
```
Comment (jfmartinz) [inline Web Development/README.md:1316]:
```suggestion
        <td><a href="https://www.youtube.com/@ApnaCollegeOfficial">Apna College</a></td>
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]

File: Web Development/README.md (hunk 1 above)
Comment (jfmartinz) [reply Web Development/README.md:1316]: Another change you need to make was the URL, change it.

Comment (SaileePhal) [reply Web Development/README.md:1316]: Ok sir will do it right a way.

Comment (SaileePhal) [reply Web Development/README.md:1316]: Done sir in #352

**Pull request closed without merging**